numba/__init__.py

```diff
@@ -219,4 +221,4 @@ def _try_enable_svml():
 # a bind to the underlying LLVM libraries which then irretrievably sets the LLVM
 # SVML state to "no SVML". See https://github.com/numba/numba/issues/4689 for
 # context.
-# ---------------------- WARNING WARNING WARNING ----------------------------
+# ---------------------- WARNING WARNING WARNING ----------------------------
```

Comment:
Please could a newline be added back in here, I think it was accidentally removed? Thanks.